src/core_plugins/region_map/public/region_map_controller.js

```diff
@@ -54,7 +54,8 @@ module.controller('KbnRegionMapController', function ($scope, $element, Private,
         return;
       }
 
-      updateChoroplethLayer($scope.vis.params.selectedLayer.url);
+      console.log($scope.vis.params.selectedLayer);
```

Comment:
how can you commit console.log statements ? pre-commit step doesn't run for you ? (i tried doing that several times but was unable to do so ... )